pkg/chains/storage/gcs/gcs.go

```diff
@@ -61,6 +67,7 @@ func (b *Backend) StorePayload(rawPayload []byte, signature string, opts config.
 	// name/namespace as well.
 	// $bucket/taskrun-$namespace-$name-$uid/$key.signature
 	// $bucket/taskrun-$namespace-$name-$uid/$key.payload
+	// Note(rgreinho): it looks like theere is no UUID in the object names :/
```

Comment:
i think i removed the uuid in the object name because it is part of `opts.Key`, which gets included.